Makefile

```diff
@@ -349,6 +349,7 @@ $(eval $(call build-package,py3-s3transfer,0.5.1-r0))
 $(eval $(call build-package,py3-urllib3,1.26.13-r0))
 $(eval $(call build-package,py3-yaml,6.0-r0))
 $(eval $(call build-package,aws-cli,1.27.38-r0))
+$(eval $(call build-package,kubevela,1.6.7-r0))
```

Comment:
One small thing here that causes pipeline to crash, you are using `1.6.7` version here but the actual package version is `1.7.0`, you need to upgrade it in here too.

Reply:
Good catch! I think we should add a check to linter for this case.

Reply:
There is already a check for this in lint.sh where I found this error.